src/group_theory/submonoid/center.lean

```diff
@@ -33,6 +33,14 @@ def center : submonoid M :=
 
 @[to_additive] lemma coe_center : ↑(center M) = set.center M := rfl
 
+@[simp]
+lemma center_to_subsemigroup : (center M).to_subsemigroup = subsemigroup.center M := rfl
+
+lemma _root_.add_submonoid.center_to_add_subsemigroup (M) [add_monoid M] :
+  (add_submonoid.center M).to_add_subsemigroup = add_subsemigroup.center M := rfl
+
+attribute [to_additive add_submonoid.center_to_add_subsemigroup] submonoid.center_to_subsemigroup
+
```

Comment:
```suggestion
```
Does this actually do anything? My understanding is that `to_additive` just adds the corresponding additive declaration, but since you anyway had to do it by hand I don't see the point.

Reply:
Yes, it tells to_additive that if it sees one definition when additivizing something else, to replace it with the other one

Reply:
Ah, I see, thanks!